spec/features/app_spec.rb

```diff
@@ -613,3 +613,16 @@
     end
   end
 end
+
+# we test the region images queries for google
+describe 'google region images query' do
+  describe 'google images' do
+    before do
+      @path = '/v1/google/europe-north1/images/active.xml'
```

Comment:
What happens if there are two regions with different active.xml? Or we don't have such scenario?

Reply:
my understanding is that the Google images are global, so there is no region metadata for them, which causes the list to come back empty if a region is specified. See https://github.com/SUSE-Enceladus/public-cloud-info-service/issues/8  , so in this case if there are multiple regions, their active images should be the same.

Reply:
There is always only 1 active image for a given image type. For CSPs where images are global (Google and Oracle) the global image names that match the filters should be returned when a specific region is queried.